javascript/chronometer.js

```diff
@@ -1,26 +1,38 @@
 class Chronometer {
   constructor() {
-    // ... your code goes here
+    this.currentTime = 0;
+    this.intervalId = 0;
+    this.currentTimeMil= 0;
+    this.intervalIdMil = 0;
   }
   startClick(callback) {
-    // ... your code goes here
+    this.intervalId = setInterval(() => {this.currentTime += 1;},1000)
+    this.intervalIdMil = setInterval(()=> {this.currentTimeMil += 1;},1)
   }
   getMinutes() {
-    // ... your code goes here
-  }
+    return Math.floor(this.currentTime/60)
+    }
   getSeconds() {
-    // ... your code goes here
+    return this.currentTime % 60;
   }
-  twoDigitsNumber() {
-    // ... your code goes here
+
+  getMilliseconds(){
+    return this.currentTimeMil.toString().slice(-2)
   }
+
+  twoDigitsNumber(number) {
+    return number.toString().length === 2 ? number.toString() : '0' + number.toString();
+    } 
+    
```

Comment:
Good work Emad!

To turn a number into a string, you can use template strings. You can also use a different condition
```
twoDigitsNumber(number) {
    return number > 10? `${number}` : `0${number}`;
    }